package.json

```diff
@@ -26,7 +26,7 @@
     "webpack-dev-server": "^3.11.2"
   },
   "engines": {
-    "node": "16.20.2",
+    "node": "20.19.0",
```

Comment:
Any points why 20.19.0? Looks like latest LTS version is 22.14.0
<img width="416" alt="Screenshot 2025-03-28 at 16 14 11" src="https://github.com/user-attachments/assets/cb9ab9b9-fe27-48ef-a275-6f68ad80d6bd" />

Reply:
Also `"@rails/webpacker": "5.4.3",` -> `"@rails/webpacker": "5.4.4",`

Reply:
One more thing, bump version:
```
    "@didww/encrypt": "1.5.2",
    "@rails/actioncable": "^6.1.3",
```